app/models/card.rb

```diff
@@ -1,2 +1,20 @@
 class Card < ActiveRecord::Base
+  validates :original_text, presence: true
+  validates :translated_text, presence: true
+  validates :review_date, presence: true
+  validate :translated_should_be_not_equal_to_original
+
+  before_validation :set_review_date
+
+  protected
+
+    def translated_should_be_not_equal_to_original
+      if original_text.downcase == translated_text.downcase
+        errors.add(:translated_text, "перевод не может быть равен начальному тексту")
+      end
+    end
+
+    def set_review_date
+      self.review_date = DateTime.now + 3
```

Comment:
DateTime.now - это дата+время, оно срабатывает и прибавляет +3 чего-то абсолютно непонятно чего, 3 секунды? 3 дня? 3 часа? 
в этот раз повезло и руби действительно прибавляет дни, так как тебе и нужно. Но это некорректный код, потому что неочевидный.
Используй крутые штуки рельсы:
`DateTime.now + 3.days`
или 
`3.days.since`